azext_iot/operations/hub.py

```diff
@@ -1345,3 +1345,56 @@ def _iot_hub_monitor_feedback(target, device_id, wait_on_id):
 
     events3 = importlib.import_module('azext_iot.operations.events3._events')
     events3.monitor_feedback(target=target, device_id=device_id, wait_on_id=wait_on_id, token_duration=3600)
+
+
+def iot_hub_distributed_tracing_show(cmd, hub_name, device_id, resource_group_name=None):
+    device_twin = _iot_hub_distributed_tracing_show(cmd, hub_name, device_id, resource_group_name)
+    return _customize_device_tracing_output(device_twin['deviceId'], device_twin['properties']['desired'],
+                                            device_twin['properties']['reported'])
+
+
+def iot_hub_distributed_tracing_update(cmd, hub_name, device_id, sampling_mode, sampling_rate,
+                                       resource_group_name=None):
+    if int(sampling_rate) not in range(0, 100):
```

Comment:
can sampling reate not be an integer?

Reply:
the only integer values are possible for sampling rate